### DISCUSSION TITLE
What happens with images in albums on deduplicate?

### DISCUSSION THREAD

**Discussion by nidomiro** [immich-app/immich#10953] [Q&A]
I'm currently in the process of migration from nextcloud to immich.
So far the experience is great.
It's a really nice tool, thanks for creating it :)

I just found the `Utilities -> Duplicates` view in the UI and was wondering what will happen when both images are the same (=duplicate) and I only want to keep one :

1.  But both are in an album. Will both albums reference the still existing image afterwards?
2. One is in an album and I choose to delete the one in the album. Will the one I keep appear in the album instead of the deleted one?

**Comment by AFumi39** [comment]
Just tried in v1.114.0
If you have 2 images and one is in 1 ore more albums, and the other one no, if you delete the one that's in the album, the other one does not inherit the albums of the deleted one.
That's strange behavior, seems obvious to me that a sort of "merge" should be done

**Comment by bo0tzz** [comment]
#10262
